web/src/components/_README.md

```diff
@@ -0,0 +1,10 @@
+# Components
+
+## Why this folder?
+
+- Add individual components in this folder. Things like FancyButton, SearchBar,
```

Comment:
Might be too vague - would specify "this is for widgets that do not take up the full page", etc. Whatever you want